statsd/container_test.go

```diff
@@ -0,0 +1,65 @@
+package statsd
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseContainerID(t *testing.T) {
+	for in, out := range map[string]string{
```

Comment:
Could we rename the variables:

```suggestion
	for input, expectedResult := range map[string]string{
```